### DISCUSSION TITLE
Track vps migrations

### DISCUSSION THREAD

**Discussion by asarubbo** [vultr/vultr-cli#295] [General]
Hello, I'm new to vultr and vultr-cli. Is there a way to display if an instance is under migration?

**Comment by optik-aper** [comment]
Hello @asarubbo and welcome to Vultr!  Will you clarify what you mean by an instance under migration?

**Comment by asarubbo** [comment]
I mean a vps moved from a physical host to another.

I'm an long time hetzner-cloud user and for example when you run `hcloud server list` in the normal condition you will see `running` under the STATUS column, while in the migration time you will see `migrating` under the STATUS column.

**Reply by optik-aper** [reply]
Ah, I understand you now.  I don't think the API exposes that detail but I'll inquire.

**Reply by optik-aper** [reply]
It looks like these are the current possible status values from the API for an instance:
```
status: active, pending, suspended, resizing
power_status:  running, stopped
server_status: none, locked, installingbooting, ok
```
It doesn't look like we expose migrating status.

**Reply by asarubbo** [reply]
Can I request to expose the migration status? If yes where is the correct place?

**Reply by optik-aper** [reply]
Certainly!  The best place would probably be to open a support ticket at https://my.vultr.com/support/

**Reply by asarubbo** [reply]
done, thanks :)
